community_tasks/french_evals.py

```diff
@@ -116,6 +128,22 @@ def prompt_gpqa_fr(line, task_name: str = None):
     trust_dataset=True,
     version=0,
 )
-
+# bac-fr task
+bac_fr_task = LightevalTaskConfig(
+    name="bac-fr",
+    suite=["community"],
+    prompt_function=prompt_bac_fr,
+    hf_repo="fr-gouv-coordination-ia/bac-fr",
+    hf_subset="default",
+    hf_avail_splits=["train"],
+    evaluation_splits=["train"],
+    few_shots_split=None,
+    few_shots_select="random_sampling",
+    generation_size=1,
+    metric=[],
```

Comment:
We need to define the metric :)
If generative, the generation_size will need to be higher than 1 token I think